src/DIRAC/Core/Base/Script.py

```diff
@@ -48,6 +48,11 @@
 gIsAlreadySetUsageMsg = False
 gIsAlreadyInitialized = False
 
+gLogger.warn(
```

Comment:
is `warn` now printed in scripts by default?
This is printed at import time, and maybe from somewhere that the user doesn't control.
It is entirely unclear what "custom configuration" is supposed to mean. `parseCommandLine` also loads the global configuration.

Reply:
Yes, better to use decorator, fixed.